core/notifications/migrations/20210316025847_setup.up.sql

```diff
@@ -0,0 +1,289 @@
+CREATE EXTENSION IF NOT EXISTS "uuid-ossp";
```

Comment:
why is there 2021 code migration?

Reply:
It’s the way to add sqlxmq. 

https://github.com/Diggsey/sqlxmq/tree/master?tab=readme-ov-file#database-schema

. Not the best thing about the library but it has been working for us reliably.